test/integration/test_bind.rb

```diff
@@ -10,7 +10,9 @@ def test_bind_timeout
     error = assert_raise Net::LDAP::Error do
       @ldap.bind(method: :simple, username: "uid=user1,ou=People,dc=rubyldap,dc=com", password: "passworD1")
     end
-    assert_equal('Connection timed out - user specified timeout', error.message)
+    msgs = ['Operation timed out - user specified timeout',
+            'Connection timed out - user specified timeout']
+    assert_send([msgs, :include?, error.message])
```

Comment:
> Up to you whether you want to do it in this PR, or follow up with a separate one.

I think next PR. I have some ideas about simplifying invocation of `Net::LDAP.new`, driven by having to wrap the whole gem more than once. Changing the exception/error message for timeouts feels like it belongs in a broader discussion about API changes for the gem.